wordpress-mobile/WordPress-Android
# Issue/13268 filters 1

Parent #13268

This PR
- adds a divider to the filter bar
- adds a shadow when the recycler view scroll position != 0
- adds border to the chips
- updates margins/paddings

Note:
- filter labels and cancel actions are coming in the next PR
- I'll ask for a design review on the next PR when the filter bar is finalized

To test:
1. Enable ActivityLogFilters feature flag
2. Open Activity Log
3. Check the filter bar and its styles
4. Scroll down and notice a shadow appears
--------------
1. Disable ActivityLogFilters feature flag
2. Open Activity Log
3. Verify the divider and the filter bar are hidden

PR submission checklist:
